samtranslator/schema/schema.json

```diff
@@ -193550,10 +193550,10 @@
         "InvokeRole": {
           "anyOf": [
             {
-              "type": "string"
+              "type": "object"
```

Comment:
I don't understand why the schema is changing.

Reply:
curious too

Reply:
no idea. in gh action it decided to generate with different order: https://github.com/aws/serverless-application-model/actions/runs/4296629078/jobs/7488579979

Reply:
possibly pydantic orders the keys according to the hash of our module `samtranslator`